### PR TITLE
I noticed the onError event was missing and therefore inbound.

### PR DESCRIPTION
**Pull request by peterhel**
I took the liberty to fix this.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "test": "npm run lint && npm run test:unit && npm run test:browser",
     "test:unit": "babel-tape-runner ./test/*.test.js* | faucet",
     "test:browser": "babel-tape-runner ./test/*.browser-test.js* | faucet",
-    "watch": "babel src --out-dir dist --watch"
+    "watch": "babel src --out-dir dist --watch",
+    "postinstall": "npm run build"
```
Comment (danmakenoise) [inline package.json]: Why did you need to put this in here?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-jw-player",
-  "version": "1.12.0",
+  "version": "1.12.2",
```
Comment (danmakenoise) [inline package.json]: You can go ahead and leave out the version changes, we will handle that as we merge things into the repo.

**Comment by danmakenoise** [comment]
Hey! Thanks for finding this bug! 

I have two comments in the code for you! Other than that looks great.

**Comment by peterhel** [comment]
Sorry about that, that wasn't in the pull request. That was for personal
testing only.

Den 25 okt. 2017 4:28 em skrev "Dan Phillips" <notifications@github.com>:

File: package.json (hunk 1 above)
Comment (peterhel) [reply package.json]: Sorry, didn't think that would be part of the pull request.

File: package.json (hunk 2 above)
Comment (peterhel) [reply package.json]: Sorry, didn't think that would be part of the pull request.

**Comment by danmakenoise** [comment]
@peterhel no problem, go ahead an patch it up and we will merge it in!

**Comment by peterhel** [comment]
Hey! Made a new pull request from a branch containing only what's relevant.

2017-10-27 21:33 GMT+02:00 Dan Phillips <notifications@github.com>:

**Comment by danmakenoise** [comment]
@ellell we should be able to close this now
